baseapp/baseapp_test.go

```diff
@@ -1489,7 +1482,10 @@ func TestCustomRunTxPanicHandler(t *testing.T) {
 	{
 		tx := newTxCounter(0, 0)
 
-		require.PanicsWithValue(t, customPanicMsg, func() { app.Deliver(aminoTxEncoder(), tx) })
+		require.PanicsWithValue(t, customPanicMsg, func() {
+			_, _, err := app.Deliver(aminoTxEncoder(), tx)
+			require.Error(t, err)
```

Comment:
Please don't. See my previous comment.

Reply:
I'm not sure if I understand the context from your previous comment. Can you clarify @alessio ?

Reply:
Here we're testing that `app.Deliver(aminoTxEncoder(), tx)` panics. Thus the line `require.Error(t, err)` should never be hit because of the panic() raised in `app.Deliver()`. Long story short: IMvHO it makes no sense to test the returned error, this is just dead code.

Reply:
Is it possible to set some // nolint for these specific cases?

Reply:
in fact, I've updated to check the err outside of `require.Panics...` context.